components/ContactInfo.vue

```diff
@@ -3,18 +3,36 @@
 		<h2>
 			Contact Us
 		</h2>
-		<p>
+		<div>
 			<a :href="'mailto:' + email">{{ email }}</a><br>
 			IDRC<br>
 			205 Richmond Street West<br>
```

Comment:
The address is missing the city and postal code. This was updated in the designs.